build-system/tasks/helpers.js

```diff
@@ -119,18 +120,17 @@ function doBuildJs(jsBundles, name, extraOptions) {
 async function bootstrapThirdPartyFrames(options) {
   const startTime = Date.now();
   const promises = [];
-  const {watch, minify} = options;
   thirdPartyFrames.forEach((frameObject) => {
     promises.push(
       thirdPartyBootstrap(frameObject.max, frameObject.min, options)
     );
   });
-  if (watch) {
+  if (options.watch) {
     thirdPartyFrames.forEach((frameObject) => {
       const watchFunc = () => {
         thirdPartyBootstrap(frameObject.max, frameObject.min, options);
       };
-      gulpWatch(frameObject.max).on(
+      watch(frameObject.max).on(
```

Comment:
Nit: Given the proximity here, it could be easy for someone to confuse `watch` and `options.watch`.

Could we continue to rename the export from `chokidar`? `{watch: fileSystemWatch}` or similar?

Reply:
Fair point. Done.